docs/source/code-generation/codegen-configuration.mdx

```diff
@@ -106,7 +106,7 @@ The properties to configure `output` are:
 "output": {
   "schemaTypes": {
     "moduleType": {
-      "swiftPackageManager": {}
+      "swiftPackage": {}
```

Comment:
It's long bugged me that we had `swiftPackageManager` as a [`ModuleType` option](https://www.apollographql.com/docs/ios/code-generation/codegen-configuration#module-type) and `swiftPackage` as a [`TestMocks` option](https://www.apollographql.com/docs/ios/code-generation/codegen-configuration#test-mocks); I'm happy these are more similar now. 😄